packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -245,9 +246,11 @@ export default function ProjectNav({
   activeNav,
   onProjectChange,
   hosts,
-  alerts = 0,
+  mode = Mode.Production,
```

Comment:
Hmm... Can't we require this to be passed into ProjectNav, and set the default only in MongoNav?

Reply:
Sure, same difference. Not sure how many assumptions we should make about props. This one seems fine though.

Reply:
Fair enough – I think for ProjectNav in particular, we can make a lot of assumptions given it's not built to be used outside of MongoNav.